src/olympia/devhub/tasks.py

```diff
@@ -129,10 +129,12 @@ def create_version_for_upload(addon, upload, channel):
 
 
 @task
-def create_initial_validation_results():
+def create_initial_validation_results(minimal_parsed_data):
     """Returns the initial validation results for the next tasks in the
     validation chain. Should only be called directly by Validator."""
     results = deepcopy(amo.VALIDATOR_SKELETON_RESULTS)
+    if minimal_parsed_data:
+        results.update(minimal_parsed_data)
```

Comment:
Maybe this should be added under `results['minimal_parsed_data']` instead of being merged directly in the main dict?

Reply:
I just did that

Reply:
I just did that